input/input.c

```diff
@@ -196,14 +196,14 @@ const struct m_sub_options input_config = {
         OPT_FLAG("input-cursor", enable_mouse_movements, 0),
         OPT_FLAG("input-vo-keyboard", vo_key_input, 0),
         OPT_FLAG("input-media-keys", use_media_keys, 0),
-#if HAVE_COCOA
-        OPT_FLAG("input-appleremote", use_appleremote, 0),
-#endif
 #if HAVE_SDL2_GAMEPAD
         OPT_FLAG("input-gamepad", use_gamepad, 0),
 #endif
         OPT_FLAG("window-dragging", allow_win_drag, 0),
         OPT_REPLACED("input-x11-keyboard", "input-vo-keyboard"),
+#if HAVE_COCOA
+        OPT_REMOVED("input-appleremote", "input-appleremote replaced by ..."),
```

Comment:
Is there something missing from this string?

Reply:
apparently yes, i was quite sure i changed that message. seems like some minor changes went missing.